playbook/app/pb_kits/playbook/pb_nav/_subtle_mixin.scss

```diff
@@ -42,6 +42,7 @@
     &[class*=_active] [class*=_link] {
       @include pb_title_4;
       color: $primary;
+      letter-spacing: normal !important;
```

Comment:
Which one wins out currently, when a user uses the letter-spacing global prop? Usually that one is what we reserve `important` for? Can you check and see what happens if they add that... https://playbook.powerapp.cloud/visual_guidelines#Typography

Reply:
You're right! Tested it with and without !important tag + with the typography token and using the important tag means the token does not apply if used. Removing the !important tag seems to still allow this css to work while also allowing for the dev using the typography letter spacing token. 

Thank you for catching that @jasperfurniss !

Reply:
No problem!